client/src/pages/Dashboard/DashboardPage.tsx

```diff
@@ -1,11 +1,394 @@
+          placeholder="New Title"
+          onChange={(e) =>
+            setState({
+              ...state,
+              editingInfo: { ...editingInfo, title: e.target.value },
+            })
+          }
+        />
+      </Form.Item>
+      <Form.Item label="Details">
+        <Input.TextArea
+          autoSize
+          value={editingInfo.details}
+          placeholder="Add details here..."
+          onChange={(e) =>
+            setState({
+              ...state,
+              editingInfo: { ...editingInfo, details: e.target.value },
+            })
+          }
+        />
+      </Form.Item>
+    </>
+  );
+
+  const rescheduleForm = (
+    <Form.Item label="Reschedule">
+      <DatePicker
+        value={moment.utc(editingInfo.date)} // moment defaults to local time, while JS's Date is UTC
+        onChange={(date, dateString) =>
+          setState({
+            ...state,
+            editingInfo: { ...editingInfo, date: new Date(dateString) },
+          })
+        }
+      />
+    </Form.Item>
+  );
+
+  const SyncModal = (
+    <Modal
+      onCancel={() => setState({ ...state, isModalVisible: false })}
+      onOk={handleModalSave}
+      visible={isModalVisible}
+      okText="Save"
+    >
+      <Form layout="vertical">
+        <Form.Item label="M8">
+          <Input value={editingSync.name} disabled />
+        </Form.Item>
+        <Form.Item label="Sync Date">
+          <Input
+            value={editingSync.upcomingSync.toLocaleDateString()}
+            disabled
+          />
+        </Form.Item>
+        {editingMode === EditModalEnum.SUMMARY ? summaryForm : rescheduleForm}
+      </Form>
+    </Modal>
+  );
+
+  const UpcomingSyncsNode = (
+    <>
+      {SyncModal}
+      <Row justify="center">
+        <Col>
+          <Title level={3}>Upcoming Syncs</Title>
+        </Col>
+      </Row>
+      <Row gutter={[24, 24]}>
+        {upcomingSyncs.map((sync) => (
+          <Col span={6} key={sync.id}>
+            <DashboardCard
+              upcomingSync={sync}
+              actions={[
+                <Popover
+                  content="Already scheduled, remove from feed"
+                  placement="bottom"
+                  key={`${sync.id}-carryout`}
+                >
+                  <CarryOutOutlined onClick={() => removeSync(sync)} />
+                </Popover>,
+                <Popover
+                  content="Decline this sync"
+                  placement="bottom"
+                  key={`${sync.id}-close`}
+                >
+                  <CloseOutlined onClick={() => removeSync(sync)} />
+                </Popover>,
+              ]}
+            />
+          </Col>
+        ))}
+      </Row>
+    </>
+  );
+
+  return (
+    <>
+      <Row>
+        <Col span={18}>
+          <div
+            style={{
+              backgroundColor: "#F0F2F5",
+              height: "150px",
+              padding: "20px",
+            }}
+          >
+            <Title>Hi, Paul</Title>
```

Comment:
we probably shouldn't hardcode the name here lol.
Also, could you make the greeting message bigger - like take up ~70% of the header area and be vertically centered?